src/q_and_a_service/routes.js

```diff
@@ -0,0 +1,8 @@
+const { Router } = require ('express');
+const controller = require ('./controller.js');
```

Comment:
You might consider destructuring what you are bringing in from './controller.js' here. Using `const {getQA} = require('./controller.js');` will allow you to simplify your function calls elsewhere in the file